.github/workflows/conformance-ipsec-e2e.yaml

```
@@ -168,7 +168,10 @@ jobs:
                                       --collect-sysdump-on-failure \
                                       --sysdump-hubble-flows-count=1000000 \
                                       --sysdump-hubble-flows-timeout=5m \
-                                      --flush-ct \
+                                      --log-code-owners \
+                                      --code-owners=${CILIUM_CLI_CODE_OWNERS_PATHS} \
+                                      --exclude-code-owners=${CILIUM_CLI_EXCLUDE_OWNERS} \
+                                      --flush-ct" \
```

Need to fold this into the relevant commit:

```suggestion
                                      --flush-ct \
```